docusaurus.config.js

```diff
@@ -25,6 +25,53 @@ module.exports = {
   favicon: 'img/meta/favicon-96x96.png',
   organizationName: 'ionic-team',
   projectName: 'ionic-docs',
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      {
+        // Will be passed to @docusaurus/plugin-content-docs (false to disable)
+        docs: {
```

Comment:
The margin between the accordion header and the icon is incorrect in RTL (LTR is correct, so maybe we need to use `margin-inline` instead): 

| production | branch |
| - | - | 
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/777a4f43-e067-4259-981f-8566211a1269) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/40183b6d-d601-4ebc-a160-77ec2e18616f) |